be/src/olap/options.h

```diff
@@ -26,14 +26,25 @@
 namespace doris {
 
 struct StorePath {
-    StorePath() : capacity_bytes(-1) { }
-    StorePath(const std::string& path_, int64_t capacity_bytes_)
-        : path(path_), capacity_bytes(capacity_bytes_) { }
+    StorePath() : capacity_bytes(-1), storage_medium(TStorageMedium::HDD) { }
+    StorePath(const std::string &path_, int64_t capacity_bytes_)
+        : path(path_),
+          capacity_bytes(capacity_bytes_),
+          storage_medium(TStorageMedium::HDD) { }
+    StorePath(const std::string &path_, int64_t capacity_bytes_,
+              TStorageMedium::type storage_medium_)
+        : path(path_),
+          capacity_bytes(capacity_bytes_),
+          storage_medium(storage_medium_) { }
     std::string path;
     int64_t capacity_bytes;
+    TStorageMedium::type storage_medium;
 };
 
-OLAPStatus parse_conf_store_paths(const std::string& config_path, std::vector<StorePath>* path);
+OLAPStatus parse_root_path(const std::string &root_path, StorePath *path);
```

Comment:
better to add some comment for this function to let other known what this function does.
```suggestion
OLAPStatus parse_root_path(const std::string& root_path, StorePath* path);
```